db/migrate/20170902213139_create_mucked_homes.rb

```diff
@@ -0,0 +1,14 @@
+class CreateMuckedHomes < ActiveRecord::Migration[5.1]
+  def change
+    create_table :mucked_homes do |t|
+      t.string :name
+      t.text :description
+      t.string :phone
+      t.string :pin
+      t.string :lat
```

Comment:
To keep it consistent in the DB, lat and lng should be written out, latitude and longitude. They should be floats not strings.